src/DynamoCore/Library/XmlDocumentationExtensions.cs

```diff
@@ -173,7 +171,7 @@ private static string GetMemberElementName(FunctionDescriptor member)
                     // parameters are listed according to their type, not their name
                     string paramTypesList = String.Join(
                         ",",
-                        member.Parameters.Select(x => x.Type.ToShortString()).Select(PrimitiveMap).ToArray()
+                        member.Parameters.Select(x => x.Type.ToString()).Select(PrimitiveMap).ToArray()
```

Comment:
Hi @aosyatnik, I just want to make sure this is absolutely safe to do. What are the differences between `ToShortString` and `ToString` here? Do you have an example for reference?